tests/posix/common/prj.conf

```diff
@@ -8,5 +8,6 @@ CONFIG_POSIX_MQUEUE=y
 CONFIG_HEAP_MEM_POOL_SIZE=4096
 CONFIG_MAX_THREAD_BYTES=4
 CONFIG_THREAD_NAME=y
+CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000
```

Comment:
I would suggest not adding this here, but maybe adding the tickless option in `testcase.yaml` as another test configuration. Technically, you could also add newlib to that same configuration and if necessary, adjust allowed platforms in that configuration.

Reply:
@cfriedt  CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000, is to fix this issue https://github.com/zephyrproject-rtos/zephyr/issues/53605, and if we add disable tickless option and we will find the issue https://github.com/zephyrproject-rtos/zephyr/issues/56163.

Reply:
@hakehuang - I believe adding `CONFIG_SYS_CLOCK_TICKS_PER_SEC=1000` to the test project is more of a workaround than a solution. I will try and adjust the testcase so that it passes for the nordic boards mentioned in 53605 and will work with you to ensure that 56163 works as well.

Reply:
@cfriedt , just a reminder, in the cortex_m_systick.c
```
uint32_t cyc = elapsed() + cycle_count - announced_cycles;
return cyc / CYC_PER_TICK;
```
so if the platform runs too fast, the default CYC_PER_TICK is 10000, which means the two timer read  will return the same value.